lib/ioki/model/operator/vehicle.rb

```diff
@@ -98,25 +103,25 @@ class Vehicle < Base
                   on:   [:create, :read, :update],
                   type: :string
 
-        attribute :seats,
-                  on:             [:create, :read, :update],
-                  omit_if_nil_on: [:create, :update],
-                  type:           :integer
-
-        attribute :storage_spaces,
-                  on:             [:create, :read, :update],
-                  omit_if_nil_on: [:create, :update],
-                  type:           :integer
-
-        attribute :walker_bays,
-                  on:             [:create, :read, :update],
-                  omit_if_nil_on: [:create, :update],
-                  type:           :integer
-
-        attribute :wheelchair_bays,
-                  on:             [:create, :read, :update],
-                  omit_if_nil_on: [:create, :update],
-                  type:           :integer
+        deprecated_attribute :seats,
```

Comment:
You can also write 
```
attribute :seats
	deprecated: true
        on:             [:create, :read, :update],
	omit_if_nil_on: [:create, :update],
	type:           :integer
```
or can't you?

Reply:
As @phylor mentioned [here](https://github.com/ioki-mobility/ioki-ruby/pull/304#issuecomment-1810483790)
> In the model classes, we define deprecated attributes using the new method deprecated_attribute instead of attribute. This justs sets deprecated: true on the attribute definition, but is more visible than supplying the option to attribute.

It is for better visibility 👍